### PR TITLE
Google Play Store search command

### LINKED ISSUE #67: Google Play search command
https://github.com/facundoolano/google-play-scraper

### PR DESCRIPTION
**Pull request by Doges**
Closes #67

Preview:

### CODE REVIEW

**Comment by pedrofracassi** [comment]
@Doges please don't request review from each team member. GitHub automatically requests review based on the codeowners file.

**Comment by Doges** [comment]
> 
> 
> @Doges please don't request review from each team member. GitHub automatically requests review based on the codeowners file.

bruh

**Review by moruzerinho6 — APPROVED** [review]
Looks good to me.

File: src/apis/GooglePlay.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  }
+
+  async searchApp (query) {
+    try {
+      return await gPlay.search({ term: query, num: 10, fullDetail: true })
+    } catch (e) {
+      return new Error('Not found')
+    }
+  }
+}
```
Comment (pedrofracassi) [inline src/apis/GooglePlay.js:18]: Wouldn't only returning the promise straight away be a better idea (without try/catching it)? We could then catch any errors with `.catch` in the command instead

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1628,5 +1628,15 @@
...
     "lastUpdated": "Last Updated at",
     "notFound": "Code not found"
+  },
+  "googleplay": {
+    "commandDescription": "Searches for an APP on Google Play Store",
+    "commandUsage": "<application name>",
+    "developer": "Developer:",
+    "price": "Price:",
+    "genre": "Genre:",
+    "rating": "Rating:",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1638]:
```suggestion
    "developer": "Developer",
    "price": "Price",
    "genre": "Genre",
    "rating": "Rating",
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

File: src/apis/GooglePlay.js (hunk 1 above)
Comment (moruzerinho6) [reply src/apis/GooglePlay.js:18]: Making every command(If another is made in the future) that uses the API having to handle itself?

Unless all the APIs work like you suggested, I don't see why make this move.

Comment (pedrofracassi) [reply src/apis/GooglePlay.js:18]: Well, we'd still have to do something with the Error you're returning. Promises already have rejection, you're just wrapping it in another layer of complexity.

Comment (pedrofracassi) [reply src/apis/GooglePlay.js:18]: Actually, this whole API file is quite redundant, as this library requires no tokens or configuration and could be used directly in the command instead

Comment (Doges) [reply src/apis/GooglePlay.js:18]: I mean, the API is kinda glitchy, I already told you in the whatsapp

**Review by pedrofracassi — APPROVED** [review]
LGTM

**Review by perronosaurio — DISMISSED** [review]

**Review by pedrofracassi — DISMISSED** [review]
LGTM

**Review by almeidx — APPROVED** [review]

**Review by moruzerinho6 — APPROVED** [review]
LGTM

**Comment by Doges** [comment]
puta

**Review by perronosaurio — APPROVED** [review]

**Comment by Doges** [comment]
corno

**Review by pedrofracassi — APPROVED** [review]

**Review by pedrofracassi — APPROVED** [review]

**Pull request merged**